packages/fast-components-react-base/src/auto-suggest/auto-suggest.tsx

```diff
@@ -87,9 +87,19 @@ class AutoSuggest extends Foundation<
     }
 
     public componentDidUpdate(prevProps: AutoSuggestProps): void {
-        const updatedMenuVisibility: boolean = this.validateMenuState(
```

Comment:
Does this mean that if you set the prop `isMenuOpen` since you are no longer calling `validateMenuState` you will override the prop?

Reply:
No, the prop always wins in the validate function.

Reply:
This check actually enables setting a prop to override existing state.

Reply:
I see, what would help next time is making it clear that the `toggleMenu` method already runs a check using the `validateMenuState` before setting the state.